templates/zerver/app/settings_overlay.html

```diff
@@ -47,14 +47,23 @@ <h1>{{ _('Settings') }}<span class="section"></span></h1>
                 <li tabindex="0" data-section="organization-profile">
                     <i class="icon fa fa-lock" aria-hidden="true"></i>
                     <div class="text">{{ _('Organization profile') }}</div>
+                    {% if not is_admin %}
+                    <i class="locked fa fa-lock" aria-hidden="true"></i>
+                    {% endif %}
                 </li>
                 <li tabindex="0" data-section="organization-settings">
                     <i class="icon fa fa-flask" aria-hidden="true"></i>
                     <div class="text">{{ _('Organization settings') }}</div>
+                    {% if not is_admin %}
+                    <i class="locked fa fa-lock" aria-hidden="true"></i>
+                    {% endif %}
                 </li>
                 <li tabindex="0" data-section="organization-permissions">
                     <i class="icon fa fa-lock" aria-hidden="true"></i>
                     <div class="text">{{ _('Organization permissions') }}</div>
+                    {% if not is_admin %}
+                    <i class="locked fa fa-lock" aria-hidden="true"></i>
+                    {% endif %}
                 </li>
                 <li tabindex="0" data-section="emoji-settings">
                     <i class="icon fa fa-smile-o" aria-hidden="true"></i>
```

Comment:
I think there should be a code block for this one, depending on a slightly more complex condition involving `realm_add_emoji_by_admins_only`